src/server/dstu2/patient/controllers/patient.controller.js

```diff
@@ -1,7 +1,9 @@
 const { ServerError } = require('../../../utils/error.utils');
 const { VERSIONS } = require('../../../../constants');
+const { Patient } = require('../../../resources/Patient');
```

Comment:
Could also pull this in from dstu2 resources instead of top level resources, tha path would be `require('../../resources')` instead